riley-research/IPSA
# Display generated annotations

1. Renamed in `Peak` interface (`mZ` to `mz`) and relevant implementations to ensure consistency.
2. Added fragment matching logic to `PeptideStore` for peak mass error calculations.
3. Enhanced `formatLabel` method in `PlotDataStore` to support Unicode superscripts and subscripts for charged ions and neutral losses, ensuring correct label formatting.

### How to test if necessary
1. Test fragment matching: Ensure peaks are correctly matched to fragments and mass errors are calculated precisely.
2. Validate Unicode formatting: Confirm that ion labels are correctly formatted with superscripts/subscripts in the `PlotDataStore` class.
